docs/4.2. Behaviour - Querying.md

```diff
@@ -4,6 +4,14 @@ _(c) AMWA 2016, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
 
 The Query API provides a read-only interface to the Nodes and their sub-resources which are currently held in the registry. Browsing of the Query API may be performed via the RESTful API, or by using the websocket protocol ([RFC 6455](https://tools.ietf.org/html/rfc6455)) which is suggested for most applications.
 
+## API Resources
+
+Where the behaviour associated with an API attribute is not sufficiently clear from its name and schema constraints, this is documented below.
+
+### Subscriptions
+
+`secure` Indicates whether the WebSocket connection is provided via an encrypted connection or not (ws:// vs. wss://). Unless otherwise indicated the value of this attribute SHOULD be 'false' if the API is being presented via HTTP, and 'true' for HTTPS. Query API clients MAY choose to include this attribute in requests for subscriptions, however they will receive a 400 response code unless the Query API explicitly supports a mismatch between encrypted HTTP and WebSocket connections.
```

Comment:
Only a suggestion:
Query API clients MAY choose to _specify the opposite value_ in requests for subscriptions...